code/modules/client/preferences_savefile.dm

```diff
@@ -181,6 +181,15 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 		if(lickable)
 			ENABLE_BITFIELD(vore_flags,LICKABLE)
 
+	if(current_version < 29)
```

Comment:
oh, yea, you need to bump this and the max_savefile_version define to 30 cuz of Putnam's vorey refactor